addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1,14 +1,23 @@
 /** @odoo-module **/
 
+import { reactive } from "@odoo/owl";
```

Comment:
usually we import the OWL stuff last, but I don't think it's a convention. Still if you think that's a good idea to have it after all "odoo" imports, would be nice if you could quickly change it.